### ISSUE TITLE
Error when placing code inside junction point (Windows)

### ISSUE THREAD

**Issue by evgenyt1** [s-panferov/awesome-typescript-loader#506]
When declaring global variable in definition file (or using `declare global`) having error:
```
ERROR in [at-loader] ..\..\..\Dropbox\_projects\temp\s1\src\some.d.ts:1:15
    TS2451: Cannot redeclare block-scoped variable 'aa'.

ERROR in [at-loader] ./src/some.d.ts:1:15
    TS2451: Cannot redeclare block-scoped variable 'aa'.
```
This happens only when having project (and running it) inside Windows junction point, that points to some other dir.
Replacing `at-loader` with `ts-loader` fixes the issue. 
Running `tsc` doesn't show these errors.

Not sure if this is also affected by Unix symlinks.

Simple to reproduce

**webpack.config.json:**

```
module.exports = {
    entry: './src/index.ts',
    output: { filename: 'bundle.js', path: __dirname + "/dist" },
    resolve: { extensions: [".ts"] },
    module: { rules: [{ test: /\.tsx?$/, loader: "awesome-typescript-loader" }] },
};
```

**index.ts:**
`/// <reference path="./some.d.ts"/>`

**some.d.ts:**
`declare const aa:string;`

Full source code: https://github.com/evgenyt1/at-loader-issue/tree/master
